storage_v2/v1adapter/trace_compare.go

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) 2017 Uber Technologies, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-package integration
+package v1adapter
```

Comment:
I don't think this move is necessary. (a) adapter is for things that translate between ptrace and model structures, and (b) I don't think this comparator is used anywhere else so no reason to move it to a general package.

Reply:
I used the Comparator in my tests, to prevent cyclic imports, I had to move it. 

Oooh, should I use `v1adapter_test` as the package for the tests instead?

Reply:
yes you can